skore/src/skore/_sklearn/_plot/metrics/precision_recall_curve.py

```diff
@@ -112,10 +88,25 @@ class PrecisionRecallCurveDisplay(_ClassifierCurveDisplayMixin, DisplayMixin):
     >>> classifier = LogisticRegression(max_iter=10_000)
     >>> report = EstimatorReport(classifier, **split_data)
     >>> display = report.metrics.precision_recall()
-    >>> display.plot(pr_curve_kwargs={"color": "tab:red"})
+    >>> display.plot(relplot_kwargs={"palette": "Set2"})
     """
 
-    _default_pr_curve_kwargs: dict[str, Any] | None = None
+    _default_relplot_kwargs: dict[str, Any] = {
+        "kind": "line",
+        "estimator": None,
+        "x": "recall",
+        "y": "precision",
```

Comment:
I would probably hard code the 4 first parameters when calling `replot`. I would reserve the kwargs for the style only.